src/libYARP_OS/include/yarp/os/Os.h

```diff
@@ -189,6 +189,20 @@ namespace yarp {
          * @since YARP 2.3.70
          */
         YARP_OS_API int fork(void);
+
+       /**
+        * @brief Toggle the OS energy saving feature
+        *
+        * This function toggle the state of the energy saving feature
+        * that may affect YARP calls
+        *
+        * For example, in macOS this function toggles the state of the AppNap
+        * feature, which put the process into an idle state.
+        *
+        * @param enable true if the energy saving mode should be enabled. False otherwise.
+        * @since YARP 2.3.72
+        */
+        YARP_OS_API void toggleEnergySavingMode(bool enable);
```

Comment:
"toggle" does not seem the correct verb to use with a boolean switch, since toggle means something that switches between on and off every time you call it...
Since I suspect we will use it only for disabling the energy saving mode, perhaps `disableEnergySavingMode(bool disable = true)` might be better?